Makefile

```diff
@@ -1,4 +1,11 @@
+.PHONY: package test
+
 package:
 	./check.sh
 	git rev-parse HEAD >resources/build.txt
 	lein do clean, test :all, package
+
+test:
+	cd test/repo/no-errors && mvn test
+	-cd test/repo/some-errors && mvn test
```

Comment:
`-` ignores the exit code, which you need because that invocation of `mvn test` will fail.